runtime/onert/core/src/compiler/train/LoweredTrainableGraph.cc

```diff
@@ -105,19 +104,18 @@ void LoweredTrainableGraph::lowerGraph(const CompilerOptions &options)
     .append(std::make_unique<compiler::pass::PermutationInsertionPass>(*this))
     .run();
 
-  // TODO Enable after merging TrainableOperationConverter
-  // // TODO Move converting Permute op into PermutationInsertionPass
-  // auto op_converter = TrainableOperationConverter{_trainable_graph, nullptr};
-  // _trainable_graph.operations().iterate(
-  //   [&](const onert::ir::OperationIndex &index, const onert::ir::IOperation &op) {
-  //     if (op.opcode() == ir::OpCode::Permute)
-  //     {
-  //       auto top = op_converter(index);
-  //       auto gen_index = _trainable_graph.replaceOperation(index, std::move(top));
-  //       UNUSED_RELEASE(gen_index);
-  //       assert(gen_index == index);
-  //     }
-  //   });
+  // TODO Move converting Permute op into PermutationInsertionPass
+  auto op_converter = TrainableOperationConverter{_trainable_graph, nullptr};
+  _trainable_graph.operations().iterate(
+    [&](const onert::ir::OperationIndex &index, const onert::ir::IOperation &op) {
+      if (op.opcode() == ir::OpCode::Permute)
+      {
+        auto top = op_converter(index);
```

Comment:
I thought `top` as it is, the opposite of bottom. : )

op_converter` coverts op to trainable operation, and push it in a stack, and returns the top index. But I realized that `top` means `trainable operation`.

Reply:
How about changing to convert operation instead of index? changing the name to `trainable_op`?

Reply:
I hope I am the only one who was confused by `top`. To make sure, `trainable_op` looks good. `t_op`, `tr_op`, ... are all good.

Reply:
OK, I will change that. What is your opinion on changing to convert operation instread of index?

Reply:
I create a new RR, #10996 for that.